source/03-specifications/03-counter-report-common-attributes-and-elements.rst

```diff
@@ -165,81 +220,152 @@ Table 3.p (below): List of Data_Type Values
        Discovery_Service\ |br|\ |lb|
        eJournal\ |br|\ |lb|
        Repository
-     - PR, DR, TR, IR\ |br|\ |lb|
-       PR_P1, DR_D1, TR_J1, TR_J2, TR_J3, TR_J4
+     - PR, DR, TR
 
    * - Multimedia
-     - Multimedia content such as audio, image, streaming audio, streaming video, and video.
+     - Multimedia content such as audio, image, streaming audio, streaming video, and video, that cannot be easily classified as a specific multimedia Data_Type.
      - Multimedia\ |br|\ |lb|
        Multimedia_Collection
-     - PR, DR, IR\ |br|\ |lb|
-       PR_P1, DR_D1, IR_M1
+     - PR, DR, IR
+
+   * - News_Item
+     - An article from a newspaper or magazine, or a news item available as a standalone content item not aggregated into a title, for example in an institutional repository.
+     - A&I_Database\ |br|\ |lb|
+       Aggregated_Full_Content\ |br|\ |lb|
+       Discovery_Service\ |br|\ |lb|
+       eJournal\ |br|\ |lb|
+       Repository\ |br|\ |lb|
+       Scholarly_Collaboration_Network
+     - PR, IR
 
    * - Newspaper_or_Newsletter
      - Textual content published serially in a newspaper or newsletter.
      - A&I_Database\ |br|\ |lb|
        Aggregated_Full_Content\ |br|\ |lb|
        Discovery_Service\ |br|\ |lb|
-       eJournal\ |br|\ |lb|
-       Repository
-     - PR, DR, TR, IR\ |br|\ |lb|
-       PR_P1, DR_D1
+       eJournal
+     - PR, DR, TR
 
    * - Other
-     - Content that cannot be classified by any of the other Data_Types.
-
-       Note that Data_Type Other MUST NOT be used if there isn’t sufficient information available to classify the content.
+     - Content that has been labelled with a data type that does not exist within and cannot be mapped to COUNTER’s Code of Practice. Other MUST NOT be used if there is not sufficient information available to classify the content.
      - A&I_Database\ |br|\ |lb|
        Aggregated_Full_Content\ |br|\ |lb|
        Discovery_Service\ |br|\ |lb|
        Repository
-     - PR, DR, TR, IR\ |br|\ |lb|
-       PR_P1, DR_D1
+     - PR, DR, TR, IR
+
+   * - Patent
+     - A patent document representing an exclusive right granted for an invention, which is a product or a process that provides, in general, a new way of doing something, or offers a new technical solution to a problem. Typically associated with a patent number.
+     - Aggregated_Full_Content\ |br|\ |lb|
+       Discovery_Service\ |br|\ |lb|
+       Full_Content_Database\ |br|\ |lb|
```

Comment:
Full_Content_Database seems wrong to me, especially since Aggregated_Full_Content is listed.

Reply:
I've checked with a publisher hosting patents and they say patents are closer to item than title, and that full content db is more relevant than aggregated. resolved per their advice